### PR TITLE
Publish alpha+beta tags on github and npm

### PR DESCRIPTION
**Pull request by taymoork2**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -173,21 +173,32 @@ jobs:
...
+
+            BUILD_PUBLIC_PATH="https://code.s4d.io/widget-recents/archives/${VERSION_NUMBER}/" npm run build:package widget-recents
+
+            BUILD_PUBLIC_PATH="https://code.s4d.io/widget-recents/archives/${VERSION_NUMBER}/" npm run build sri widget-recents
+
+            BUILD_BUNDLE_PUBLIC_PATH="https://code.s4d.io/widget-recents/archives/${VERSION_NUMBER}/"
+
+            BUILD_PUBLIC_PATH="https://code.s4d.io/widget-recents/archives/${VERSION_NUMBER}/demo/" npm run build:package widget-recents-demo
+
+            BUILD_PUBLIC_PATH="https://code.s4d.io/widget-demo/archives/${VERSION_NUMBER}/" npm run build:package widget-demo
```
Comment (InteractiveTimmy) [inline .circleci/config.yml]: This section was re-written to fix an issue with the live widget demos and the fix does not appear to be present in the changes. See [this line](https://github.com/webex/react-widgets/pull/1021/files#diff-1d37e48f9ceff6d8030570cd36286a61L189) as an example in which both ENV vars are specified in one line, which allows for the widgets to successfully point to the correct script build file during the build process.

File: .circleci/config.yml [hunk 2]
```diff
@@ -173,21 +173,32 @@ jobs:
...
-            BUILD_PUBLIC_PATH="https://code.s4d.io/widget-space/archives/${VERSION}/" npm run build:package widget-space
-            BUILD_PUBLIC_PATH="https://code.s4d.io/widget-space/archives/${VERSION}/" npm run build sri widget-space
-            BUILD_BUNDLE_PUBLIC_PATH="https://code.s4d.io/widget-space/archives/${VERSION}/" BUILD_PUBLIC_PATH="https://code.s4d.io/widget-space/archives/${VERSION}/demo/" npm run build:package widget-space-demo
-            BUILD_PUBLIC_PATH="https://code.s4d.io/widget-recents/archives/${VERSION}/" npm run build:package widget-recents
-            BUILD_PUBLIC_PATH="https://code.s4d.io/widget-recents/archives/${VERSION}/" npm run build sri widget-recents
-            BUILD_BUNDLE_PUBLIC_PATH="https://code.s4d.io/widget-recents/archives/${VERSION}/" BUILD_PUBLIC_PATH="https://code.s4d.io/widget-recents/archives/${VERSION}/demo/" npm run build:package widget-recents-demo
-            BUILD_PUBLIC_PATH="https://code.s4d.io/widget-demo/archives/${VERSION}/" npm run build:package widget-demo
+
```
Comment (adamweeks) [inline .circleci/config.yml]: Rebase your changes on @InteractiveTimmy's fixes in this block.

File: .circleci/config.yml (hunk 1 above)
Comment (taymoork2) [reply .circleci/config.yml]: Hmmm I did rebase, guess I need to manually “revert”

File: .circleci/config.yml [hunk 3]
```diff
@@ -258,11 +269,24 @@ jobs:
...
           from: ~/react-widgets/packages/node_modules/@ciscospark/widget-demo/dist
           to: s3://${AWS_BUCKET}/widget-demo/archives/${VERSION_NUMBER}
+      - aws-s3/copy:
+          from: ~/react-widgets/.version
+          to: s3://${AWS_BUCKET}/widget-space/alpha
+      - aws-s3/copy:
+          from: ~/react-widgets/.version
+          to: s3://${AWS_BUCKET}/widget-recents/alpha
+      - aws-s3/copy:
+          from: ~/react-widgets/.version
```
Comment (adamweeks) [inline .circleci/config.yml]: This path doesn't look right

**Comment by adamweeks** [comment]
Closing as we are not going to currently go this route with npm publishing.

**Pull request closed without merging**